spread.yaml

```diff
@@ -671,8 +673,11 @@ debug-each: |
 
     # Keep it as the last step in debug-each
     echo '# tasks executed on system'
-    # since the runs file does not have a newline at EOF, add one 
-    echo "" | cat "$RUNTIME_STATE_PATH/runs" - || true
+    if [ -f "$RUNTIME_STATE_PATH/runs" ]; then
+        echo "EXECUTED_TESTS=$(cat $RUNTIME_STATE_PATH/runs)"
```

Comment:
I don't see the `EXECUTED_TESTS` variable anywhere in the snapd repo, is this purely aesthetic or is the variable set in the log output for a particular reason?

Reply:
thanks for the review, I need a way to query the debug output to extract the list of tests, so I added this `EXECUTED_TESTS` var works as a tag.